win32/Lib/win32rcparser.py

```diff
@@ -226,6 +226,7 @@ def currentNumberToken(self):
     # quote.  consumes all tokens until the end of the string
     def currentQuotedString(self):
         # Handle quoted strings - pity shlex doesn't handle it.
+        assert self.token
```

Comment:
Improves the error message in the case where `self.token` is `None` (from an `AttributeError` to this `AssertionError`)